src/index.html

```diff
@@ -17,61 +17,335 @@
+          <span class="product-card__code-value">195434</span>
+        </p>
+        <div class="product-card__rating">
+          <div class="block-stars block-stars--4">
+            <div class="block-stars__stars"></div>
+            <div class="block-stars__stars"></div>
+            <div class="block-stars__stars"></div>
+            <div class="block-stars__stars"></div>
+            <div class="block-stars__stars"></div>
+          </div>
+          <p class="product-card__reviews">
+            <span class="product-card__reviews-title">Reviews:</span>
+            <span class="product-card__reviews-value">5</span>
+          </p>
+        </div>
+        <p class="product-card__price">
+          <span class="product-card__price-title">Price:</span>
+          <span class="product-card__price-value">$2,199</span>
+        </p>
+        <a 
+          href="#" 
+          class="product-card__link" 
+          data-qa="hover"
+        >buy
+        </a>
       </div>
-      <div class="catalog__card">
-        Product card
+      <div class="product-card" data-qa="card">
+        <img 
+          class="product-card__img" 
+          src="images/imac.jpeg" 
+          alt="product"
+        >   
+        <h2 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="product-card__code">
+          <span class="product-card__code-title">Product code:</span>
+          <span class="product-card__code-value">195434</span>
+        </p>
+        <div class="product-card__rating">
+          <div class="block-stars block-stars--4">
+            <div class="block-stars__stars"></div>
+            <div class="block-stars__stars"></div>
+            <div class="block-stars__stars"></div>
+            <div class="block-stars__stars"></div>
+            <div class="block-stars__stars"></div>
+          </div>
+          <p class="product-card__reviews">
+            <span class="product-card__reviews-title">Reviews:</span>
+            <span class="product-card__reviews-value">5</span>
+          </p>
+        </div>
+        <p class="product-card__price">
+          <span class="product-card__price-title">Price:</span>
+          <span class="product-card__price-value">$2,199</span>
+        </p>
+        <a 
+          href="#" 
+          class="product-card__link" 
+          data-qa="hover"
+        >buy
+        </a>
       </div>
-      <div class="catalog__card">
-        Product card
+      <div class="product-card" data-qa="card">
+        <img 
+          class="product-card__img" 
+          src="images/imac.jpeg" 
+          alt="product"
+        >   
+        <h2 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="product-card__code">
+          <span class="product-card__code-title">Product code:</span>
+          <span class="product-card__code-value">195434</span>
+        </p>
```

Comment:
Do you style them separately? No needs. Check all over the page and apply changes, please
```suggestion
        <p class="product-card__code">Product code:195434</p>
```